test/server.js

```diff
@@ -0,0 +1,58 @@
+import {SocketConfig} from '../src/config.js'
+import {DebateManager} from "../src/debatemanager.js";
+import io from 'socket.io-client'
+import request from 'request'
+import chai from 'chai';
+
+const expect = chai.expect;
+const should = chai.should();
+
+describe('Server connection test', () => {
+    const debateManager = new DebateManager();
+    const srvAddress = `http://localhost:${SocketConfig.SOCKET_PORT}`;
+
+    before(() => {
+        debateManager.start();
+    });
+
+    it('Server response body', (done) => {
+        request(srvAddress , (err, req, body) => {
+            expect(body).to.equal('Socket.io server');
+            done();
+        });
+    });
+
+    describe('socket.io connection test', () => {
+        let client;
+
+        beforeEach(() => {
+            client = io.connect(srvAddress, {
+                path: '/socket-io',
```

Comment:
You can use `static DEFAULT_PATH = '/socket-io';` from SocketConfig class.
```suggestion
                path: '${SocketConfig.DEFAULT_PATH}',
```